### PR TITLE
Create Stop-SPOTenantRename

### PR DESCRIPTION
**Pull request by WayneEwington**
Documenting the new PowerShell cmdlets to support SPO Tenant Renames.

Please include this PR with the other files:
* Start-SPOTenantRename
* Get-SPOTenantRenameStatus
* Stop-SPOTenantRename

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+# Stop-SPOTenantRename
+
+## SYNOPSIS
+
+ [!IMPORTANT]
+> - This feature is available only for customers who have been accepted into the Private Preview program. [Request to join the Private Preview](https://aka.ms/SPOTenantRenameNomination). Note that submitting a nomination does not guarantee acceptance into the program. We review the nominations on a case-by-case basis and will contact you if you're accepted.
+> - This feature is not available for organizations that have set up multi-geo.
+
+Cancels the scheduled job to change the SharePoint domain name for your orgnization in Microsoft 365.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename:21]:
```suggestion
Cancels the scheduled job to change the SharePoint domain name for your organization in Microsoft 365. 
```

File: sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## PARAMETERS
+
+### -Reason
+
+Specifies the reason why the job is being canceled.
+
+```yaml
+Type: String
+Parameter Sets: Default
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename:54]: If there is only one param set and it is not named, this is probably just `Parameter Sets: (All)`

**Review by get-itips — CHANGES_REQUESTED** [review]
The file is missing the extension ".md"

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips and @tseward

File: sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename (hunk 2 above)
Comment (get-itips) [reply sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename:54]: @WayneEwington could you see my comment here? cc @yogkumgit

**Comment by WayneEwington** [comment]
Renaming the file to include .md has "confused" this Pull Request, so closing it and creating a new one.

**Pull request closed without merging**